dotnet-authserver/src/TeacherIdentity.AuthServer/Services/DqtApi/TeacherInfo.cs

```diff
@@ -7,4 +7,6 @@ public record TeacherInfo
     public required string LastName { get; init; }
     public required DateOnly DateOfBirth { get; init; }
     public required string? NationalInsuranceNumber { get; init; }
+    public bool PendingNameChange { get; init; }
+    public bool PendingDateOfBirthChange { get; init; }
```

Comment:
```suggestion
    public required bool PendingNameChange { get; init; }
    public required bool PendingDateOfBirthChange { get; init; }
```

Reply:
Are we worried about these fields not currently being returned from the DQT API? We would be throwing an exception every time we called Dqt

Reply:
Oh wow, I didn't realise the JSON serializer actually cared about `required`. Definitely leave as-is then!